src/main/java/org/matsim/run/LausitzScenario.java

```diff
@@ -138,8 +157,17 @@ protected Config prepareConfig(Config config) {
 			throw new RuntimeException(e);
 		}
 
-		// TODO: recreate counts format with car and trucks
 
+		if (emissions == EmissionAnalysisHandling.PERFORM_EMISSIONS_ANALYSIS) {
+//		set hbefa input files for emission analysis
+			EmissionsConfigGroup eConfig = ConfigUtils.addOrGetModule(config, EmissionsConfigGroup.class);
+			eConfig.setDetailedColdEmissionFactorsFile(HBEFA_FILE_COLD_DETAILED);
+			eConfig.setDetailedWarmEmissionFactorsFile(HBEFA_FILE_WARM_DETAILED);
```

Comment:
If you only use AVERAGE, you **should** be able to run it without the detailed tables.